_posts/2023-11-08-routines.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Share your best Trino SQL routine"
+author: "Martin Traverso, Dain Sunstrom, David Phillips, Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/trino-sql-routine.png
+show_pagenav: false
+---
+
+We want to see the best [SQL routines]({{site_url}}/docs/current/routines.html)
+you can write, feature them as [examples in the
+documentation]({{site_url}}/docs/current/routines/examples.html), and send you
+some goodies as a reward!
+
+<!--more-->
+
+With the recent [Trino 431
+release]({{site_url}}/docs/current/release/release-431.html) we shipped a
+feature that has been awaited by many Trino users for a long, long time. [SQL
+routines]({{site_url}}/docs/current/routines.html) are an easy way to define our
+own procedural, custom functions. All users on your Trino instance can then use
+that function in their queries and enjoy the new feature to simplify their
+queries.
+
+The new process of writing a routine in your client tool in SQL replaces the old
+way of having to create a custom plugin in Java, compiling it, and getting the
```

Comment:
We aren't replacing the old way.  The old was was really inaccessible to all but the most technical users.

Reply:
Updated